src/main/resources/com/nirima/jenkins/plugins/docker/DockerTemplate/template.jelly

```diff
@@ -84,6 +84,10 @@
             <f:checkbox/>
         </f:entry>
 
+        <f:entry title="${%Run slave in exclusive mode}" field="exclusiveMode">
+            <f:checkbox/>
+        </f:entry>
```

Comment:
Will check later, but probably we should reuse existed dropdown control that already has clever description.